app/javascript/chat/chat.jsx

```diff
@@ -1878,7 +1860,7 @@ export default class Chat extends Component {
 
   renderChannelBackNav = () => {
     return (
-      <button
+      <Button
         className="crayons-btn crayons-btn--icon-rounded crayons-btn--ghost active-channel__back-btn"
         onClick={this.navigateToChannelsList}
         onKeyUp={(e) => {
```

Comment:
@reobin or @link2twenty, this could be refactored in a future PR to use the new Keyboard component/hook.